ios/StatusIm/Info.plist

```diff
@@ -58,6 +58,7 @@
 	<string>We need to access your photo storage to give you an ability to select photos</string>
 	<key>UIAppFonts</key>
 	<array>
+		<string>RobotoMono-Medium.ttf</string>
```

Comment:
why do we want to use Roboto on iOS, i know maybe it's a question to @andmironov